### PR TITLE
certbot: add Dutch translation

### PR DESCRIPTION
**Pull request by leonvsc**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

**Review by kbdharun — APPROVED** [review]
The translation LGTM, Thanks for your contribution. 

I think we can leave the domain as `.com` itself.

File: pages.nl/linux/certbot.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# certbot
+
+> De Let's Encrypt Agent om automatisch TLS certificaten te verkrijgen en te vernieuwen.
+> Opvolger van `letsencrypt`.
+> Meer informatie: <https://certbot.eff.org/docs/using.html>.
+
+- Verkrijg een nieuw certificaat via webroot authorisatie, maar installeer het certificaat niet automatisch:
+
+`sudo certbot certonly --webroot --webroot-path {{pad/naar/webroot}} --domain {{subdomein.voorbeeld.nl}}`
```
[the remark this answers is not in the document]
Comment (leonvsc) [reply pages.nl/linux/certbot.md]: Changed them back to .com

File: pages.nl/linux/certbot.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+- Verkrijg een nieuw certificaat via nginx authorisatie, installeer het nieuwe certificaat automatisch:
+
+`sudo certbot --nginx --domain {{subdomein.voorbeeld.nl}}`
```
[the remark this answers is not in the document]
Comment (leonvsc) [reply pages.nl/linux/certbot.md]: Changed them back to .com

File: pages.nl/linux/certbot.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+- Verkrijg een nieuw certificaat via apache authorisatie, installeer het nieuwe certificaat automatisch:
+
+`sudo certbot --apache --domain {{subdomein.voorbeeld.nl}}`
```
[the remark this answers is not in the document]
Comment (leonvsc) [reply pages.nl/linux/certbot.md]: Changed them back to .com

File: pages.nl/linux/certbot.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+
+- Vernieuw alle Let's Encrypt certificaten die binnen 30 dagen verlopen (vergeet achteraf niet alle servers te herstarten die dit certificaat gebruiken):
+
+`sudo certbot renew`
+
+- Simuleer het verkrijgen van een nieuw certificaat, maar sla deze niet op, op een harde schijf:
+
+`sudo certbot --webroot --webroot-path {{pad/naar/webroot}} --domain {{subdomein.voorbeeld.nl}} --dry-run`
```
[the remark this answers is not in the document]
Comment (leonvsc) [reply pages.nl/linux/certbot.md]: Changed them back to .com

File: pages.nl/linux/certbot.md [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+
+- Verkrijg een onvertrouwd test certificaat:
+
+`sudo certbot --webroot --webroot-path {{pad/naar/webroot}} --domain {{subdomein.voorbeeld.nl}} --test-cert`
```
[the remark this answers is not in the document]
Comment (leonvsc) [reply pages.nl/linux/certbot.md]: Changed them back to .com

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**